docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+
+Exercise Groups
+^^^^^^^^^^^^^^^
+- Artemis exam mode allows you to define multiple exercise variants so that each student can receive a unique exam. This is done through the exercise groups. Exercise groups represent an individual exercise slot for each student exam. Within one exercise group you can define different exercises. 
+- Artemis selects one exercise per exercise group randomly to generate the individual student exams. 
+- You can distinguish between ``mandatory`` exercise groups and ``non-mandatory`` exercise groups. By default, every exercise group is mandatory. You can set the ``mandetory`` flag when you add a exercise group initially, or later by clicking ``edit`` on the exercise group.
+
+.. figure:: instructor/exercise_variants.png
+   :alt: Exercise Groups with different Exercise Variants
+   :align: center
+
+   Exercise Groups with different Exercise Variants
+
+Add Exercises
+^^^^^^^^^^^^^
+- Exercises are grouped by exercise groups. For every student exam, one exercise per exercise group will be chosen, see `Exercise Groups`_. 
+
+     - **Hint:** If you want all student to have the same exam, define only one exercise per exercise group.
+
+- To add exercises navigate to the *Exercise Groups* of the exam. On the header of each exercise group you will find the available exercise types. You can choose between ``creating a new exercise`` or ``importing an existing one`` from your courses. 
+
+    - **Hint:** For programming exercises you can check the option to ``allow manual assessment``. If you do not set this flag, your assessors will not be able to manually assess the student's submissions during the :ref:`assessment process <exam_assessment>`.
+
+- For exercise types ``text``, ``programming``, and ``modeling``, you can also define example submissions and example assessments to guide your assessor team.
+
+.. figure:: instructor/add_exercises.png
+   :alt: Add different Exercises
+   :align: center
+
+   Add different Exercises
+
+Registering Students
+^^^^^^^^^^^^^^^^^^^^
+- To register students to the exam, navigate from the exam management to the *Students* page. Here you are presented with two options to register students. You can: 
+
+    1. Add students manually my searching via the search bar
+    2. Bulk import students using a ``.csv`` file. You can do this by pressing the ``Import students`` button.
+
+- You can also remove students from the exam. When you do so, you have the option to also delete their participations and submissions linked to the user’s student exam. 
+    
+    - **Hint:** Just registering the students to the exam will not allow them to participate in the exam. First, individual student exams must be generated. For more information see `Manage Student Exams`_.
+
+.. figure:: instructor/add_students.png
+   :alt: Register Students
+   :align: center
+
+   Register Students Page
+
+Manage Student Exams
+^^^^^^^^^^^^^^^^^^^^
+-  The student exams are managed via the *Student Exams* page. 
```

Comment:
Please start with an explanation what a student exam is: it consists of an individual set of exercises (based on the configured exercise groups and exercises). After all exercises are configured properly, Artemis generates an individual student exam with a random set of exercises for each registered student